src/deletepy/__init__.py

```
@@ -0,0 +1,209 @@
+    block_user,
+    delete_user,
+    get_user_details,
+    get_user_email,
+    get_user_id_from_email,
+    revoke_user_grants,
+    revoke_user_sessions,
+    unlink_user_identity,
+)
+
+# Utilities
+from .utils import (
```

Flake8 (7.2.0)

[error] redefinition of unused 'FileOperationError' from line 19

(F811)